### ISSUE TITLE
`toil --version` freezes

### ISSUE THREAD

**Issue by hannes-ucsc** [BD2KGenomics/toil#1094]
The backtrace is nuts. A simple `toil --version` shouldn't involve importing job stores implementations. I'm also wondering why importing the AWSJobStore decends into all this encryption stuff.

```
(venv) mesosbox@ip-172-31-24-114:~$ toil --version
^C


Traceback (most recent call last):
  File "/usr/local/bin/toil", line 9, in <module>
    load_entry_point('toil==3.4.0a1.dev227', 'console_scripts', 'toil')()
  File "/usr/local/lib/python2.7/dist-packages/toil/utils/toilMain.py", line 8, in main
    modules = loadModules()
  File "/usr/local/lib/python2.7/dist-packages/toil/utils/toilMain.py", line 35, in loadModules
    from toil.utils import toilKill, toilStats, toilStatus, toilClean
  File "/usr/local/lib/python2.7/dist-packages/toil/utils/toilStatus.py", line 28, in <module>
    from toil.leader import ToilState
  File "/usr/local/lib/python2.7/dist-packages/toil/leader.py", line 34, in <module>
    from toil.jobStores.aws.jobStore import AWSJobStore
  File "/usr/local/lib/python2.7/dist-packages/toil/jobStores/aws/jobStore.py", line 62, in <module>
    import toil.lib.encryption as encryption
  File "/usr/local/lib/python2.7/dist-packages/toil/lib/encryption/__init__.py", line 16, in <module>
    from _nacl import *
  File "/usr/local/lib/python2.7/dist-packages/toil/lib/encryption/_nacl.py", line 16, in <module>
    from nacl.secret import SecretBox
  File "/usr/local/lib/python2.7/dist-packages/nacl/secret.py", line 18, in <module>
    import nacl.bindings
  File "/usr/local/lib/python2.7/dist-packages/nacl/bindings/__init__.py", line 17, in <module>
    from nacl.bindings.crypto_box import (
  File "/usr/local/lib/python2.7/dist-packages/nacl/bindings/crypto_box.py", line 17, in <module>
    from nacl._lib import lib
  File "/usr/local/lib/python2.7/dist-packages/nacl/_lib/__init__.py", line 32, in <module>
    ffi = FFI()
  File "/usr/local/lib/python2.7/dist-packages/cffi/api.py", line 52, in __init__
    from . import cparser, model
  File "/usr/local/lib/python2.7/dist-packages/cffi/cparser.py", line 2, in <module>
    from .commontypes import COMMON_TYPES, resolve_common_type
  File "/usr/local/lib/python2.7/dist-packages/cffi/commontypes.py", line 9, in <module>
    from _cffi_backend import _get_common_types
KeyboardInterrupt
```

**Comment by hannes-ucsc** [comment]
Might have been fixed along with #1115 (#1117).

**Comment by hannes-ucsc** [comment]
The backtrace shows the precise import that caused #1115, so I'm assuming that this is fixed.
